pkg/pinecone/config/seed/component.json

```diff
@@ -0,0 +1,99 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Pinecone Connector Component Spec",
+  "type": "object",
+  "properties": {
+    "task": {
+      "title": "Task",
+      "enum": [
+        "QUERY",
+        "UPSERT"
+      ],
+      "default": "QUERY"
+    }
+  },
+  "allOf": [
+    {
+      "if": {
+        "properties": {
+          "task": {
+            "const": "QUERY"
+          }
+        }
+      },
+      "then": {
+        "type": "object",
+        "required": [
+          "topK"
+        ],
+        "properties": {
+          "namespace": {
+            "type": "string",
+            "description": "The namespace to query"
+          },
+          "topK": {
+            "type": "integer",
+            "description": "The number of results to return for each query"
+          },
+          "includeValues": {
+            "type": "boolean",
+            "description": "Indicates whether vector values are included in the response",
+            "default": false
+          },
+          "includeMetadata": {
+            "type": "boolean",
+            "description": "Indicates whether metadata is included in the response as well as the ids",
+            "default": false
+          },
+          "vectors": {
```

Comment:
the name should be `vector`, not `vectors`

Reply:
updated